packages/@expressive-code/plugin-collapsible-sections/src/styles.ts

```diff
@@ -0,0 +1,87 @@
+import { StyleSettings, ExpressiveCodeTheme, ResolvedCoreStyles, setAlpha } from '@expressive-code/core'
+
+export const collapsibleSectionClass = 'ec-section'
+
+export const collapsibleSectionsStyleSettings = new StyleSettings({
+	closedBorderWidth: '0',
+	closedPadding: '4px 0 4px var(--padding-inline)',
+	closedMargin: '0',
+	closedFontFamily: 'inherit',
+	closedFontSize: 'inherit',
+	closedLineHeight: 'inherit',
+	closedTextColor: 'inherit',
+	closedBackgroundColor: ({ theme }) => setAlpha(theme.colors['editor.foldBackground'], 0.2) || 'rgb(84 174 255 / 20%)',
+	closedBorderColor: ({ theme }) => setAlpha(theme.colors['editor.foldBackground'], 0.5) || 'rgb(84 174 255 / 50%)',
+	openBorderWidth: '1px',
+	openPadding: '0',
+	openMargin: '0',
+	openBackgroundColor: 'transparent',
+	openBorderColor: 'transparent',
+})
+
+export function getCollapsibleSectionsBaseStyles(
+	theme: ExpressiveCodeTheme,
+	coreStyles: ResolvedCoreStyles,
+	styleOverrides: Partial<typeof collapsibleSectionsStyleSettings.defaultSettings>
+) {
+	const styles = collapsibleSectionsStyleSettings.resolve({
+		theme,
+		coreStyles,
+		styleOverrides,
+	})
+	const result = `
+		.${collapsibleSectionClass} {
+			& summary {
+				/* hide the default <details> marker */
+				&::marker {
+					display: inline-block;
+					content: "";
```

Comment:
Safari does not support `content` for `::marker`: https://bugs.webkit.org/show_bug.cgi?id=204163